src/components/dialog/__definition__.js

```diff
@@ -21,7 +21,13 @@ const definition = new Definition('dialog', Dialog, {
   },
   propValues: {
     title: 'Example Title for a Dialog',
-    children: '<Form>This is an example of a dialog.</Form>'
+    children: `<Form><DateInput name="date" label="Date" /><DateInput name="date" label="Date" /><DateInput name="date" label="Date" /><DateInput name="date" label="Date" />
+    <Textbox label="Name" validations={[ new PresenceValidation() ]}/>
+    <Textbox label="Name" validations={[ new PresenceValidation() ]}/>
+    <Textbox label="Name" validations={[ new PresenceValidation() ]}/>
+    <Textbox label="Name" validations={[ new PresenceValidation() ]}/>
+    <Textbox label="Name" validations={[ new PresenceValidation() ]}/>
+    <Textbox label="Name" validations={[ new PresenceValidation() ]}/>This is an example of a dialog.</Form>`
```

Comment:
I'd either reset this, or tidy it up for the demo (give the inputs different names, make the it look a little organised)

Reply:
Yes this was just done to give me a quick example for dev purposes. We will talk about the demo at the end as you suggested.